hadoop-ozone/ozone-manager/src/test/java/org/apache/hadoop/ozone/om/request/bucket/TestOMNonS3BucketCreateRequest.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ *  with the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package org.apache.hadoop.ozone.om.request.bucket;
+
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.UUID;
+
+import org.apache.hadoop.ozone.om.exceptions.OMException;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+import org.junit.Test;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.mockito.Mockito.when;
+
+/**
+ * Tests TestOMNonS3BucketCreateRequest class,
+ * which handles CreateNonS3Bucket request.
+ */
+@RunWith(Parameterized.class)
+public class TestOMNonS3BucketCreateRequest extends TestOMBucketCreateRequest {
```

Comment:
By extending `TestOMBucketCreateRequest` and making this `Parameterized`, all test cases from parent class are run for all parameters.  That's unnecessary, those parent test cases are independent of the parameters.

```
[INFO] Tests run: 40, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 11.343 s - in org.apache.hadoop.ozone.om.request.bucket.TestOMNonS3BucketCreateRequest
```

If you want to keep it parameterized, the new test should either extend `TestBucketRequest`, or be added as nested class in `TestOMBucketCreateRequest`.

I suggest getting rid of `Parameterized` run, adding test cases explicitly (as described in comment below).  If you follow that, the test cases can be moved into `TestOMBucketCreateRequest` as non-nested, top-level methods.

Reply:
good idea! Totally makes sense, updated!